src/app/feedback-widget/feedback-widget.component.ts

```diff
@@ -13,8 +13,8 @@ export class FeedbackWidgetComponent implements OnInit {
   ngOnInit() {
   }
 
-  send(comment){
-    this.state.sendFeedback(comment);
+  send(comment, username){
+    this.state.sendFeedback({comment,username});
```

Comment:
If input is empty, it'll display an error but still send the feedback, or am I missing somethnig?